pom.xml

```diff
@@ -933,7 +933,7 @@
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
 
         <!-- Carbon kernel version -->
-        <carbon.kernel.version>4.10.22</carbon.kernel.version>
+        <carbon.kernel.version>4.10.24</carbon.kernel.version>
```

Comment:
Need to update the kernel version to the new released version with the changes in https://github.com/wso2/carbon-kernel/pull/4104

Reply:
Updated to the latest kernel release 4.10.26[1]

[1] https://github.com/wso2/carbon-kernel/tree/v4.10.26